app/templates/admin/admin.html

```diff
@@ -47,8 +47,11 @@ <h3>Announcements</h3>
           {%if not ann %}<h4>No active announcements</h4>
           {%else%}
           <div class="post">
-            <a href="{{url_for('do.deleteannouncement')}}" class="btn btn-red">Nuke!</a> |
-            <a href="{{url_for('site.view_post_inbox', pid=ann.pid)}}">{{ann.title}}</a> by {{ann.user}}, <relative-time datetime="{{ann.posted.isoformat()}}Z"></relative-time>
+            <form method="POST" id="deleteannouncement" action="{{ url_for('do.deleteannouncement') }}">
+              {{ csrf_form.csrf_token() }}
+              <input id="deleteannouncement-button" type="submit" value="Nuke!" class="pure-button">
```

Comment:
```suggestion
              <input id="deleteannouncement-button" type="submit" value="Remove" class="pure-button">
```
This would make it more clear what this button does.

Reply:
Yes definitely. Would it be worth wrapping it with a call to `_` (gettext) for translation? (I’m not sure how that works.)

Reply:
The whole template needs its strings wrapped in `_()` for translation, which would deserve another PR.